go/test/action.yml

```diff
@@ -1,12 +1,21 @@
 name: 'go-test'
 description: 'Custom GitHub action that runs all Golang tests within a repository and saves the test + coverage reports as artifacts'
 
+inputs:
+  SHORT:
+    description: Set to true if you want to run all tests, including long running tests.
+    required: false
```

Comment:
Shouldn't this be flipped? When `short` is set to `true` it should SKIP long running tests

Reply:
lol yes! But this actually mirrors the `-short` argument for `go test`. I have no idea why they did it this way.

Reply:
Does it? https://pkg.go.dev/testing#hdr-Skipping
This reads like I understood it, if set (true) skip tests

Reply:
https://pkg.go.dev/testing#Short

Reply:
`short` is just a flag that returns true. You can use it either to add or skip tests. Practically speaking there will be fewer integration (long) tests than short ones, so it seems like it's more common to use this to mark integration tests and skip them if the flag isn't turned on.

Reply:
...nope, I completely misunderstood this. [https://blog.dharnitski.com/2019/04/29/skip-tests-in-go/](https://blog.dharnitski.com/2019/04/29/skip-tests-in-go/) This seems really weird. Let me have another go at this.

Reply:
Maybe using tags and build comments is a better approach.